build.gradle

```
@@ -57,6 +57,13 @@ allprojects {
 
 	group = "net.fabricmc.fabric-api"
 
+	sourceSets {
+		testmod {
+			compileClasspath += main.compileClasspath
```

Hmm you declared another compile configuration below
```suggestion
			compileClasspath += main.compileClasspath + main.output
```
~~Should compile against the rest of the main code as well~~

---

Whats the use of this?

---

not really useful as you already have implementation configuratiom